PowerEditor/src/WinControls/shortcut/shortcut.rc

```diff
@@ -19,49 +19,49 @@
 #include "shortcutRc.h"
 
 #ifndef IDC_STATIC
-#define IDC_STATIC	-1
+#define IDC_STATIC -1
 #endif
 
-IDD_SHORTCUT_DLG DIALOGEX 0, 0, 180, 105
+IDD_SHORTCUT_DLG DIALOGEX 0, 0, 180, 97
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
 EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
 CAPTION "Shortcut"
 FONT 8, TEXT("MS Shell Dlg"), 400, 0, 0x1
 BEGIN
-    CONTROL         "CTRL",IDC_CTRL_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,30,33,10
-    CONTROL         "ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,51,28,10
-    CONTROL         "SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,57,42,35,10
-    COMBOBOX        IDC_KEY_COMBO,110,40,64,270,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON      "OK",IDOK,32,74,50,14
-    PUSHBUTTON      "Cancel",IDCANCEL,103,73,50,14
-    LTEXT           "+",IDC_STATIC,45,42,8,8
-    LTEXT           "+",IDC_STATIC,96,42,8,8
-    EDITTEXT        IDC_NAME_EDIT,56,9,118,14,ES_AUTOHSCROLL
-    LTEXT           "Name :",IDC_NAME_STATIC,15,12,34,8,0,WS_EX_RIGHT
-	LTEXT           "This will disable the accelerator",IDC_WARNING_STATIC,6,93,170,8
-	LTEXT           "CONFLICT FOUND!",IDC_CONFLICT_STATIC,6,93,170,8
+    RTEXT           "&Name:",IDC_NAME_STATIC,15,9,34,8
+    EDITTEXT        IDC_NAME_EDIT,56,8,118,12,ES_AUTOHSCROLL
+    CONTROL         "&CTRL",IDC_CTRL_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,27,33,10
+    CONTROL         "&ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,48,28,10
+    CONTROL         "&SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,57,39,35,10
+    COMBOBOX        IDC_KEY_COMBO,110,37,64,270,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON      "OK",IDOK,39,66,50,14
+    PUSHBUTTON      "Cancel",IDCANCEL,95,66,50,14
+    LTEXT           "+",IDC_STATIC,45,39,8,8
+    LTEXT           "+",IDC_STATIC,96,39,8,8
+    LTEXT           "This will disable the accelerator",IDC_WARNING_STATIC,6,84,170,8
+    LTEXT           "CONFLICT FOUND!",IDC_CONFLICT_STATIC,6,84,170,8
 END
 
-IDD_SHORTCUTSCINT_DLG DIALOGEX 0, 0, 286, 114
+IDD_SHORTCUTSCINT_DLG DIALOGEX 0, 0, 282, 115
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
 EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
 CAPTION "Shortcut"
 FONT 8, TEXT("MS Shell Dlg"), 400, 0, 0x1
 BEGIN
-    CONTROL         "CTRL",IDC_CTRL_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,27,33,10
-    CONTROL         "ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,48,28,10
-    CONTROL         "SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,159,39,35,10
+    RTEXT           "&Name:",IDC_NAME_STATIC,117,9,34,8

```

Comment:
Same here:
Could you move it on the right and make it wider for the translation?

Reply:
See resolution with later commit.